core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -132,6 +140,7 @@ private[spark] class CompressedMapStatus(
  */
 private[spark] class HighlyCompressedMapStatus private (
     private[this] var loc: BlockManagerId,
+    private[this] var _stageAttemptId: Int,
     private[this] var numNonEmptyBlocks: Int,
```

Comment:
Not your code, but it doesn't look like there is any reason for this to be a `var`.